build/ci/phase-template.yml

```
@@ -11,7 +11,7 @@ phases:
       _phaseName: ${{ parameters.name }}
       _arch: ${{ parameters.architecture }}
     queue:
-      timeoutInMinutes: 40
+      timeoutInMinutes: 45
```

Why move from `40` to only `45`? If it's dying at 40, do we think the extra 12.5% allowed runtime will be enough?

How about we first set a higher limit then see how much time it will take:
```suggestion
      timeoutInMinutes: 60
```

---

It used to be 60 before, we can change it again if needed, but the ones I looked at yesterday were almost finished before cancelling.